src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/ReferenceItemVisitor.cs

```diff
@@ -136,6 +137,99 @@ public override void VisitPointerType(IPointerTypeSymbol symbol)
             });
         }
 
+        public override void VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
```

Comment:
Function pointer types are also new and look, most often like
`delegate*<void>`, `delegate* unmanaged<void>`, or `delegate* unmanaged[Stdcall]<void>`

The contents of `<>` are `<parameter, parameter, parameter, returnType>` where `returnType` must always be specified.

Reply:
Notably, it looks like this is typed as `delegate* unmanaged<System.Void>`, which I'm not sure is "desirable" since it isn't "legal" syntax, even though `System.Void` is the qualified name.